.github/script/create-release-tag.sh

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env bash
+
+######################################################################
+# An script helps us trigger the create-release workflow
+#
+# How to use it
+# ./create-release-tag.sh --release-tag 1.0.0
+#
+######################################################################
+
+# The script is a local script file. So we can not use Github-secrets,
+# And this token is the key to access the Github-API, should be maintained by the manager.
+GITHUB_TOKEN="xxx"
```

Comment:
Please help to generate a Github token with workflow permission to trigger this workflow.